gkr_iop/src/gkr/layer/zerocheck_layer.rs

```diff
@@ -340,7 +338,7 @@ fn prove_rotation<E: ExtensionField>(
             .map(|rotation_expr| match rotation_expr {
                 (Expression::WitIn(source_wit_id), _) => rotation_next_base_mle(
                     &bh,
-                    &wit.wits[*source_wit_id as usize],
+                    &wit.0[*source_wit_id as usize],
```

Comment:
nitpick: it seems we can implement `Index` trait for it, so as to use `&wit[*source_wit_id as usize]` directly

Reply:
Sure, it's certainly better! Fixed in https://github.com/scroll-tech/ceno/pull/970/commits/5682bd995729ad270e9a4995fbf3f6880ad9ec0b.